src/program/lwaftr/tests/subcommands/config_test.py

```diff
@@ -27,6 +29,26 @@
 LISTEN_SOCKET_PATH = '/tmp/snabb-lwaftr-listen-sock-%s' % DAEMON_PROC_NAME
 MANAGER_SOCKET_PATH = '/var/run/snabb/by-name/%s/config-leader-socket' % DAEMON_PROC_NAME
 
+def randomString(n=8):
```

Comment:
this should be called `random_string` (as per PEP-8 https://www.python.org/dev/peps/pep-0008/#function-and-variable-names)